### ISSUE TITLE
Invalid timestamp string

### ISSUE THREAD

**Issue by fmerhout** [tidyverse/haven#246]
Hi tidyversed,

I am having a similar issue as #216 and #230. Similarly, using an earlier version of _haven_ I was able to read in my dta file without a problem. After installing 1.0.0 I am getting the error `Invalid timestamp string (length=17): 06 okt 2014 14:00`.

I have tried the github version but the issue persists. From reading the other issues, I assume it might be the lacking capitalization or the German abbreviation.

I am really hoping to resolve this with a current _haven_ version because the previous version that worked did not support `as_factor` for dataframes.

Thanks!

**Comment by evanmiller** [comment]
Hi, do you know what software/version produced the DTA file? The lower-case German timestamp appears to be unusual.

**Comment by fmerhout** [comment]
Hi @evanmiller, 

Unfortunately I do not know which Stata version/software produced the file but I had a similar hunch and simply opened and saved it with Stata13 which resolved the time stamp issue for me. Of course that might not be feasible for everyone.
 
The file I had an issue with is a 2014 country level file for the [European Social Survey](http://www.europeansocialsurvey.org/data/). Sorry for being so vague but I am having a hard time myself determining right now from where on the website I downloaded the file. For reference purposes,  I have uploaded it [here](https://www.dropbox.com/s/z91e0yh6i54vc7i/cntry-2014_F1.dta?dl=0).

Thanks!
